src/LocalDate.js

```diff
@@ -1671,13 +1671,13 @@ export class LocalDate extends ChronoLocalDate{
      * @return {string} a string representation of this date, not null
      */
     toString() {
-        var dayString, monthString, yearString;
+        let dayString, monthString, yearString;
 
-        var yearValue = this.year();
-        var monthValue = this.monthValue();
-        var dayValue = this.dayOfMonth();
+        const yearValue = this.year();
+        const monthValue = this.monthValue();
```

Comment:
could be replaced with access to the "private" field instead of the function call?